config_example.yml

```diff
@@ -32,3 +32,9 @@ runner_pool: []
 extra_runner_timer:
   minutes: 10
   hours: 0
+
+# Define the credentials to connect your redis database
+redis:
+  host: redis
+  port: 6379
+  password: password
```

Comment:
Those config contain "secrets" I rather that you move them all into the container env and interpret them as secret in your helm templates.